bip-0001.mediawiki

```diff
@@ -67,7 +67,7 @@ Each BIP should have the following parts:
 
 * Abstract -- a short (~200 word) description of the technical issue being addressed.
 
-* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example) or licensed under the Open Publication License.
+* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example).
```

Comment:
@laanwj Currently the copyright suggestions by BIP1 are a complete mess. I think we agree that OPL should be removed and at least BSD 2-clause be added, so we don't have to delete BIPs.

However, as we are touching this line anyway, we might want to provide an alternative for the public domain rule as well. As pointed out earlier, placing a work into the public domain is not possible (or even prohibited) in some countries. (c.f. http://opensource.stackexchange.com/a/1372)
Thus, choosing a license such as CC0, WTFPL, UNLINCENSE or similar is the legally "proper" way of placing a work in the public domain. To the best of my knowledge, CC0 seems to be the preferred way of placing something in the public domain.

Right now, adding CC0 to the acceptable licenses by BIP1 is a noop, but it would solve the issue in the future where a BIP author just wants to place the BIP into public domain and thus has to dual-license as BSD 2-clause and CC0, because just CC0 is not accepted by BIP1.

(I understand that my suggestion to add CC-BY-SA may be controversial, and I don't care too much, so we could just leave this for "later")